bip-tap.mediawiki

```diff
@@ -212,12 +212,13 @@ asset ID. A given Taproot Asset tree is composed of two nested MS-SMT instances:
 
 The root hash of an asset tree, observing [[../master/bip-0341.mediawiki|BIP-341]] is
 represented as a Tapscript tree that commits to a single unique leaf of:
-* <code>tagged_hash("TapLeaf", leaf_version || taproot_asset_marker || taproot_asset_version || asset_tree_root)</code>
+* <code>tagged_hash("TapLeaf", leaf_version || taproot_asset_marker || asset_tree_root || sum_value)</code>
```

Comment:
Can also change `leaf_version` to `taproot_asset_version`? I think leaf_version originally meant the tapscript leaf version, but in practice we're always using the base leaf version.

Reply:
Yeah good point, I think that was part of the confusion. I think the most precise definition here is one that would have _both_ the leaf version and the taproot asset version.

Reply:
Yes. Both versions need to be in there. The `leaf_version` is *not* part of the script. So the first field is `taproot_asset_marker` followed by `taproot_asset_version`.